thumbor/routers/__init__.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# thumbor imaging service
+# https://github.com/thumbor/thumbor/wiki
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/mit-license
+# Copyright (c) 2011 globo.com thumbor@googlegroups.com
+
+BUILTIN_ROUTERS = [
+    "thumbor.routers.healthcheck.HealthcheckRouter",
+    "thumbor.routers.upload.UploadRouter",
```

Comment:
is it a good time to move this into it's own repo?

Reply:
Yes! :) If you feel like doing it i'm all for it. I plan on contributing with thumbor-aws to get it up and running before, though.

Reply:
Let's sync up on aws stuff in slack. I had some thoughts.